sugarloaf/src/context/mod.rs

```diff
@@ -67,13 +67,19 @@ impl Context {
         let caps = surface.get_capabilities(&adapter);
 
         // TODO: Fix formats with signs
-        let unsupported_formats = wgpu::TextureFormat::Rgba8Snorm;
+        // FIXME: On Nvidia GPUs usage Rgba16Float texture format causes driver to enable HDR.
+        // Reason for this is currently output color space is poorly defined in wgpu and 
+        // anything other than Srgb texture formats can cause undeterministic output color
+        // space selection which also causes colors to mismatch. Optionally we can whitelist 
+        // only the Srgb texture formats for now until output color space selection lands in wgpu. See #205
+        // TODO: use output color format for the CanvasConfiguration when it lands on the wgpu
+        let unsupported_formats = vec![wgpu::TextureFormat::Rgba8Snorm, wgpu::TextureFormat::Rgba16Float];
```

Comment:
I often use Rgba16Float in macos/metal, since this issue is only reproduce-able on windows my suggestion will be update it for:

```rust
#[cfg(windows)]
let unsupported_formats = vec![wgpu::TextureFormat::Rgba8Snorm, wgpu::TextureFormat::Rgba16Float];

#[cfg(not(windows))]
let unsupported_formats = vec![wgpu::TextureFormat::Rgba8Snorm];
```

Reply:
It seems like there is an is_srgb filter on TextureFormat if you want I can do that filter on Windows

Reply:
I think `is_srgb` is not properly working in 0.16.0 but was fixed in 0.17.0, although I can't update wgpu crate yet before testing in few devices.